queries/kotlin/highlights.scm

```diff
@@ -405,15 +408,15 @@
 ; NOTE: `interpolated_identifier`s can be highlighted in any way
 (line_string_literal
 	"$" @punctuation.special
-	(interpolated_identifier) @none)
+	(interpolated_identifier) @variable)
```

Comment:
We need to look why `@none` no longer works. Actually this is supposed to be `@none` and `@variable`. If `@variable` has no highlight set then the string is supposed to be highlighted as `NONE` (reset the string highlighting for the interpolated range.

So
```
(interpolated_identifier) @none @variable)
```